ipsum-threat-intelligence-feed/info.json

```diff
@@ -0,0 +1,94 @@
+{
+  "name": "ipsum-threat-intelligence-feed",
+  "version": "1.0.0",
+  "label": "IPsum Threat Intelligence Feed",
+  "description": "IPsum is a threat intelligence feed based on 30+ different publicly available lists of suspicious and/or malicious IP addresses. This connector facilitates automated operations such as indicators.<br/><br/>This connector has a dependency on the <a href=\"/content-hub/all-content/?contentType=solutionpack&amp;tag=ThreatIntelManagement\" target=\"_blank\" rel=\"noopener\">Threat Intel Management Solution Pack</a>. Install the Solution Pack before enabling ingestion of Threat Feeds from this source.",
```

Comment:
Description could be better. e.g.: This connector facilitates automated operations related to fetching the list indicators .